index.js

```diff
@@ -1033,8 +1033,8 @@ module.exports = function (log, indexesPath) {
     } else console.error('Unknown type', op)
   }
 
-  function traverseEqualsAndIncludes(operation, fn) {
-    function traverseMore(ops) {
+  function forEachLeafOperationIn(operation, fn) {
```

Comment:
I renamed this because nowadays we support EQUAL, INCLUDES, PREDICATE, ABSENT, so the full name would be `traverseEqualsAndIncludesAndPredicateAndAbsent` which is pretty horrible, so I went with this one instead.

Reply:
:)